### PR TITLE
doc(portal): add documentation for auth0 dcr metadata TDX-3342

### PR DESCRIPTION
**Pull request by HenriPro**
### Description

 

Adds documentation around auth0 metadata, and how to use it via an action.  https://konghq.atlassian.net/browse/TDX-3342

### Testing instructions

tdx-3338-auth0-app-metadata featureflag must be on. This is currently enabled for all on Dev, and kongers on Prod.

Preview link: https://deploy-preview-6096--kongdocs.netlify.app/konnect/dev-portal/applications/dynamic-client-registration/auth0/

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/konnect/dev-portal/applications/dynamic-client-registration/auth0.md [hunk 1]
```diff
@@ -10,6 +10,14 @@ content_type: how-to
 * Enterprise {{site.konnect_short_name}} account.
 * An [Auth0 account](https://auth0.com/)
 
+{:.note}
+> **Note:** When using Auth0 DCR for Dev Portal, each application in auth0 will have the following metadata. This can be viewed via the auth0 dashboard, or accessed from the auth0 api.
+
+> * **konnect_portal_id**: ID of the Portal the application belongs to
+> * **konnect_developer_id**: ID of the developer in the Dev Portal
+> * **konnect_org_id**: ID of the Dev Portal the application belongs to
```
[the remark this answers is not in the document]
Comment (HenriPro) [reply app/konnect/dev-portal/applications/dynamic-client-registration/auth0.md:18]: yes

File: app/konnect/dev-portal/applications/dynamic-client-registration/auth0.md [hunk 2]
```diff
@@ -10,6 +10,14 @@ content_type: how-to
...
+> * **konnect_application_id**: ID of the application in the Dev Portal
```
[the remark this answers is not in the document]
Comment (HenriPro) [reply app/konnect/dev-portal/applications/dynamic-client-registration/auth0.md:19]: yes

File: app/konnect/dev-portal/applications/dynamic-client-registration/auth0.md [hunk 3]
```diff
@@ -10,6 +10,14 @@ content_type: how-to
 * Enterprise {{site.konnect_short_name}} account.
 * An [Auth0 account](https://auth0.com/)
 
+{:.note}
+> **Note:** When using Auth0 DCR for Dev Portal, each application in auth0 will have the following metadata. This can be viewed via the auth0 dashboard, or accessed from the auth0 api.
```
Comment (c3pko) [inline app/konnect/dev-portal/applications/dynamic-client-registration/auth0.md:14]: capitalize Auth0 and API

**Review by acgoldsmith — COMMENTED** [review]
Mainly formatting changes recommended.

**Review by acgoldsmith — APPROVED** [review]
LGTM

**Comment by acgoldsmith** [comment]
@c3pko , this PR is ready for you to merge.

**Comment by acgoldsmith** [comment]
@c3pko I've set this to DNM again while we're in the Summit release window. Will let you know when we're finished those and will clean up the conflicts that will occur before I give you the green light again.

**Pull request merged**